chronos/apps/http/httpserver.nim

```diff
@@ -65,7 +65,7 @@ type
 
   HttpProcessCallback* =
     proc(req: RequestFence): Future[HttpResponseRef] {.
-      gcsafe, raises: [].}
+      async: (raises: [CancelledError, HttpResponseError]), gcsafe.}
```

Comment:
this change breaks nimbus - to restore backwards compatibility, a `UnsafeHttpProcessCallback` with the old definition needs to be added and overloads need to be added in the places that use it .. see https://github.com/status-im/nim-chronos/blob/28a100b1350d52668ce7b9c9ae8159fdc0acace2/chronos/transports/stream.nim#L124